wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -1036,11 +1064,14 @@ public void onEvent(SyncProgressEvent event) {
             TextView syncPercentageView = findViewById(R.id.sync_status_percentage);
 
             syncPercentageView.setText(percentage + "%");
-            if (percentage == 100) {
+            syncComplete = (percentage == 100);
+            if (syncComplete) {
                 syncPercentageView.setTextColor(getResources().getColor(R.color.success_green));
                 syncStatusTitle.setText(R.string.sync_status_sync_title);
                 syncStatusMessage.setText(R.string.sync_status_sync_completed);
                 showSyncPane(R.id.sync_status_pane, false);
+                syncComplete = true;
```

Comment:
Can we remove this line?

Reply:
Yeah, it doesn't make sense ;P
Strange that Lint didn't warn me.
Removed, thanks!